RicardoMorato/CaesarCipher
# Feature/back end

### Back-end && Cipher Algorithm
**What I did:**
- Made the back-end logic and routing.
- Added the Caesar's cipher logic.

**How to test:**
- Make a `post  to the endpoint `/cipher` with a message and the offset you want to encrypt the message with. Check if the response is valid.

Resolves #7, resolves #3
